repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/contract/ContractPreProcessor.java

```diff
@@ -1,9 +1,5 @@
 package org.apache.atlas.repository.store.graph.v2.preprocessor.contract;
```

Comment:
Overall I feel that having separate API specific for migration will simplify things from dev, test & review perspective.
Let us discuss in case you can think of any challenges with API approach.

Reply:
No challenge as such. But this flow is not in use at any place. And even if users find the loophole and update the contract, it's not a security issue as they are doing it intentionally. We are moving away from this business logic to allow contract updates to versions. So that is the reason why I thought to add this. Else if you see any major concern we can go via custom API as well (users can also find out about the custom API if they can find this header approach).

Reply:
Also, if we move with a custom API, we will have to copy most of the bulk API codes to the API so that it can mimic the same behaviour.